Packs/PrismaCloud/ReleaseNotes/1_8_0.md

```diff
@@ -0,0 +1,58 @@
+
+#### Classifiers
+##### Prisma Cloud App - Classifier
+- Updated classifier for new Azure Storage playbooks
+##### Prisma Cloud - Classifier
+- Updated classifier for new Azure Storage playbooks
+##### RedLock
+- Updated for new Azure Storage playbooks
+##### prismaCloud_app
+- Updated for new Azure Storage playbooks
+
+#### Incident Fields
+- **Subscription Updated On**
+- **Subscription Description**
+- **Resource API Name**
+- **Subscription Created On**
+- **Subscription Updated By**
+- **Prisma Cloud Status**
```

Comment:
- **Prisma Cloud Status** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud Status** Associated types.